### PR TITLE
Create repo improvements

### LINKED ISSUE #4140: Improve create repository experience
* Fix "Learn more" links.
* Improved images in the docs for creating the repositories.
* Improve text in the docs for creating a repository.

### PR DESCRIPTION
**Pull request by johnnyaug**
Fixes #4140

### CODE REVIEW

File: docs/setup/create-repo.md [hunk 1]
```diff
@@ -16,41 +16,53 @@ It can be considered the lakeFS analog of a bucket in an object store. Since it
...
+   <span class="tooltip">(<a href="#">what are the other options?</a>)
+     <span class="tooltiptext">
+       The other options can help you integrate your existing tools with lakeFS.
+     </span>
+   </span>
+
+   ![Create Repository]({{ site.baseurl }}/assets/img/create_repo_s3.png){: style="padding:20px 40px"}
 
-1. Fill in a repository name.
+4. Fill in a repository name.
```
Comment (itai-david) [inline docs/setup/create-repo.md:55]: I would leave it as 1. 1. 1. or at least fix the missing 3.  even though markdown doesnt really care

File: docs/setup/create-repo.md [hunk 2]
```diff
@@ -16,41 +16,53 @@ It can be considered the lakeFS analog of a bucket in an object store. Since it
 
 ## Create the first user
 
-When you first open the lakeFS UI, you will be asked to create an initial admin user.
```
Comment (itai-david) [inline docs/setup/create-repo.md:19]: I think this should stay, in addtition to the Note below. It is much more welcoming

File: docs/setup/create-repo.md [hunk 3]
```diff
@@ -16,41 +16,53 @@ It can be considered the lakeFS analog of a bucket in an object store. Since it
...
+Note: If you already have lakeFS credentials, log in and skip to [creating the repository](#create-the-repository).
+{: .note .pb-3 }
```
Comment (itai-david) [inline docs/setup/create-repo.md:20]: Definitely belongs here, rather than after the setup :+1:

File: docs/setup/create-repo.md (hunk 1 above)
Comment (johnnyaug) [reply docs/setup/create-repo.md:55]: Oops, mistake

File: docs/setup/create-repo.md (hunk 2 above)
Comment (johnnyaug) [reply docs/setup/create-repo.md:19]: Done

**Review by itai-david — APPROVED** [review]
Thank you @johnnyaug 

Images and texts looks excellent.
Some minor comments inside - nothing crucial.

**Pull request merged**